srcCpp/PerftestTransport.cxx

```diff
@@ -649,64 +726,69 @@ std::string PerftestTransport::helpMessageString()
-<< "\t                                Values:\n"
-<< "\t                                    UDPv4\n"
-<< "\t                                    UDPv6\n"
-<< "\t                                    SHMEM\n"
-<< "\t                                    TCP\n"
-<< "\t                                    TLS\n"
-<< "\t                                    DTLS\n"
-<< "\t                                    WAN\n"
-<< "\t                                Default: UDPv4\n"
-<< "\t-nic <ipaddr>                 - Use only the nic specified by <ipaddr>.\n"
-<< "\t                                If not specified, use all available\n"
-<< "\t                                interfaces\n"
-<< "\t-transportVerbosity <level>   - Verbosity of the transport\n"
-<< "\t                                Default: 0 (errors only)\n"
-<< "\t-transportServerBindPort <p>  - Port used by the transport to accept\n"
-<< "\t                                TCP/TLS connections <optional>\n"
-<< "\t                                Default: 7400\n"
-<< "\t-transportWan                   Use TCP/TLS across LANs and Firewalls.\n"
-<< "\t                                Default: Not Set, LAN mode.\n"
-<< "\t-transportPublicAddress <ip>  - Public IP address and port (WAN address\n"
-<< "\t                                and port) (separated with ‘:’ ) related\n"
-<< "\t                                to the transport instantiation. This is\n"
-<< "\t                                required when using server mode.\n"
-<< "\t                                Default: Not Set.\n"
-<< "\t-transportWanServerAddress <a>- Address where to find the WAN Server\n"
-<< "\t                                Default: Not Set (Required)\n"
-<< "\t-transportWanServerPort <p>     Port where to find the WAN Server.\n"
-<< "\t                                Default: 3478.\n"
-<< "\t-transportWanId <id>          - Id to be used for the WAN transport.\n"
-<< "\t                                Default: Not Set (Required).\n"
-<< "\t-transportSecureWan           - Use WAN with security.\n"
-<< "\t                                Default: False.\n"
-<< "\t-transportCertAuthority <file>- Certificate authority file <optional>\n"
-<< "\t                                Default: \""
-<< TRANSPORT_CERTAUTHORITY_FILE << "\"\n"
-<< "\t-transportCertFile <file>     - Certificate file <optional>\n"
-<< "\t                                Default (Publisher): \""
-<< TRANSPORT_CERTIFICATE_FILE_PUB << "\"\n"
-<< "\t                                Default (Subscriber): \""
-<< TRANSPORT_CERTIFICATE_FILE_SUB << "\"\n"
-<< "\t-transportPrivateKey <file>   - Private key file <optional>\n"
-<< "\t                                Default (Publisher): \""
-<< TRANSPORT_PRIVATEKEY_FILE_PUB << "\"\n"
-<< "\t                                Default (Subscriber): \""
-<< TRANSPORT_PRIVATEKEY_FILE_SUB << "\"\n";
-    return oss.str();
+    << "\t===================== Transport Specific Options ======================\n"
```

Comment:
make sure that in the documentation is clear that command line parameter will overwrite the XML